src/test/data/JsonSerializableWellNusTest/invalidStudentWellNus.json

```diff
@@ -0,0 +1,44 @@
+{
+  "students": [ {
+    "name": "Hans Muster",
+    "phone": "9",
+    "address": "4th street"
+  } ],
+  "appointments" : [ {
+    "name" : "Bernice Yu",
+    "date" : "2023-10-15",
+    "startTime" : "10:00",
+    "endTime" : "11:00",
+    "description" : "Third Session"
+  }, {
+    "name" : "David Li",
+    "date" : "2023-10-16",
+    "startTime" : "14:45",
+    "endTime" : "15:45",
+    "description" : "Check-up"
+  }, {
+    "name" : "Irfan Ibrahim",
+    "date" : "2023-10-17",
+    "startTime" : "11:20",
+    "endTime" : "12:20",
+    "description" : "Follow-up"
+  }, {
+    "name" : "Roy Balakrishnan",
+    "date" : "2023-10-18",
+    "startTime" : "09:30",
+    "endTime" : "10:30",
+    "description" : "Check-up"
+  }, {
+    "name" : "Alex Yeoh",
```

Comment:
Maybe don't need Appointment since this is testing for invalid `Student` due to invalid Phone number. Or can make the Appointments match the Students.

Reply:
i think should just add to simulate an actual wellnus data file, just in case it breaks anything, i will make the appointments match the students